site/docs/v1/tech/themes/index.adoc

```diff
@@ -90,6 +115,11 @@ This page is rendered when a user clicks the URL from the verification email and
 +
 [uri]#/email/verify#
 
+[field]#Index# [required]#Required# [since]#Since 1.26.0#::
+This is the root landing page for FusionAuth.
++
+[uri]#/#
+
 [field]#OAuth authorize# [required]#Required#::
 This is the main login page for FusionAuth and is used for all interactive OAuth2 and OpenID Connect workflows.
 +
```

Comment:
"Verify registration re-send" is called "Verify registration send" in the docs, should update the docs.

Reply:
Okay, I'm not finding any additional instances in `fusionauth-site` of the phrase "Verify registration send".  
If I'm mistaken, please let me know.

Reply:
In the themes page is the only one I've seen.